mkt/constants/regions.py

```diff
@@ -108,12 +108,22 @@ class RESTOFWORLD(REGION):
     if country.get('ratingsbody'):
         country['ratingsbody'] = getattr(ratingsbodies, country['ratingsbody'])
 
+    # This if check can be removed once marketplace-constants is updated.
```

Comment:
Is marketplace-constants changing `mcc` to `mccs`? I'm not sure what has to change to remove this.